### PR TITLE
Ignore empty input to prevent an error being printed

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

**Comment by kseo** [comment]
Change the description to "Ignore empty input to prevent an error being printed".

File: src/Scheme/Evaluator.hs [hunk 1]
```diff
@@ -258,7 +258,9 @@ replLoop = do
     let quitCondition Nothing = True
         quitCondition (Just "quit") = True
         quitCondition _ = False
-    until_ quitCondition (getInputLine "Lisp>>> ") (liftIO . evalAndPrint env . fromJust)
+    let consumeLine "" = return ()
```
Comment (kseo) [inline src/Scheme/Evaluator.hs]: `evalLine` would be a better name

**Comment by kseo** [comment]
Would you resolve the conflict and upload the patch again?

**Comment by majecty** [comment]
이 패치가 없어도 이미 빈 라인을 잘 처리합니다.
닫을게요.

**Pull request closed without merging**